src/lib/tvgPicture.cpp

```diff
@@ -97,4 +97,14 @@ const uint32_t* Picture::data() const noexcept
     if (pImpl->loader) return pImpl->loader->pixels();
 
     return pImpl->pixels;
-}
\ No newline at end of file
+}
+
+
+Result Picture::paint(unique_ptr<Paint> paint) noexcept
+{
+    if (pImpl->paint) return Result::InsufficientCondition;
+    Paint* p = paint.release();
```

Comment:
please use "auto" instead of the explicit data-type
Please keep coding convention though your whole patch code.